### PR TITLE
Feature/address formatter

### PR DESCRIPTION
**Pull request by AdamJamesNaylor**

### CODE REVIEW

File: src/GoatTrip.Web/AddressInput.html [hunk 1]
```diff
@@ -45,7 +38,7 @@
 
 
 <script type="text/javascript">
-    var baseUrl = "https://locationservices.azure-api.net";
+    var baseUrl = "http://localhost:55359/";
```
Comment (tomaxworthy) [inline src/GoatTrip.Web/AddressInput.html]: Can you revert this back to the live URL

File: src/GoatTrip.RestApi/Services/AddressFormatter.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿
+namespace GoatTrip.RestApi.Services {
+    public class AddressFormatter {
```
Comment (tomaxworthy) [inline src/GoatTrip.RestApi/Services/AddressFormatter.cs]: What does this do? Happy to be missing something.

Comment (AdamJamesNaylor) [inline src/GoatTrip.RestApi/Services/AddressFormatter.cs]: Yep, that class is spurious. Will remove and repush.

File: src/GoatTrip.RestApi/Services/LocationModelMapper.cs [hunk 3]
```diff
@@ -0,0 +1,99 @@
+﻿namespace GoatTrip.RestApi.Services {
+    using System.Collections.Generic;
+    using DAL.DTOs;
+    using Models;
+
+    public class LocationModelMapper
```
Comment (tomaxworthy) [inline src/GoatTrip.RestApi/Services/LocationModelMapper.cs]: Suggest writing some more tests for this class - as could be re-factored in future, and has a lot of business logic

**Pull request merged**